doc/src/guides/solving.md

```diff
@@ -0,0 +1,45 @@
+| [ Solve a system of equations algebraically ](http://docs.sympy.org)              | [  $x^2 + y = 2, x - y = 4$  ](http://docs.sympy.org)                                                                                              |
+| [ Solve an equation numerically ](http://docs.sympy.org)                          | [ $cos(x) = x $ ](http://docs.sympy.org)                                                                                                     |
+| [ Solve a partial differential equation algebraically ](http://docs.sympy.org)    | [${\frac {\partial ^{2}(x^3 y^3)}{\partial x\partial y}}=9 $](http://docs.sympy.org)                                                                     |
+| [ Solve an ordinary differential equation algebraically  ](http://docs.sympy.org) | [ $f\prime\prime(x) + 9f(x)=0 $ ](http://docs.sympy.org)                                                    |
+| [ Solve a system of linear equations algebraically ](http://docs.sympy.org)       | [  $x + y = 2, x - y = 0$  ](http://docs.sympy.org)                                                                                           |
+| [ Solve a system of nonlinear equations algebraically ](http://docs.sympy.org)    | [  $x^2 + y^3 = 1, x^3 - y^2 = 0 $  ](http://docs.sympy.org)                                                                                      |
+| [ Solve a matrix problem algebraically ](http://docs.sympy.org)                   | [ $ \left[\begin{array}{cc} 1 & 1\\1 & -1\end{array}\right] \left[\begin{array}{cc} x\\y\end{array}\right] = \left[\begin{array}{cc} 2\\0\end{array}\right] $ ](http://docs.sympy.org) |
+| [ Solve an inequality algebraically ](http://docs.sympy.org)                      | [ $ x^2 < 4 $ ](http://docs.sympy.org)                                                                                                        |
+| [ Solve a system of inequalities algebraically ](http://docs.sympy.org)           | [ $ x^2 < 4, x > 0 $ ](http://docs.sympy.org)                                                                                                 |
+| [ Solve a polynomial algebraically ](http://docs.sympy.org)                       | [ $ x^2 - x = 0 $ ](http://docs.sympy.org)                                                                                                |
+| [ Solve a diophantine equation algebraically ](http://docs.sympy.org)             | [ $x^2 - 4xy + 8y^2 - 3x + 7y - 5 = 0 $ ](http://docs.sympy.org)                                                                                 |
+| [ Evaluate an integral symbolically ](http://docs.sympy.org)                      | [ $\int 2x\,dx 	$ ](http://docs.sympy.org)                                                                                                          |                                                                |
```

Comment:
I don't really think this belongs here.

Reply:
Here are the search queries in the top 1000 that involve integral:

Top queries | Clicks | Impressions | CTR | Position
-- | -- | -- | -- | --
integral python | 82 | 5931 | 1.38% | 9.28
python integral | 53 | 4522 | 1.17% | 9.8
sympy integral | 2222 | 3163 | 70.25% | 1.03
sympy definite integral | 568 | 1151 | 49.35% | 1.08
integral sympy | 727 | 1035 | 70.24% | 1
indefinite integral python | 210 | 721 | 29.13% | 1.77
sympy double integral | 220 | 704 | 31.25% | 1.79
python indefinite integral | 194 | 665 | 29.17% | 1.73
integral python sympy | 305 | 453 | 67.33% | 1.01
definite integral sympy | 202 | 399 | 50.63% | 1.03
sympy integrals | 246 | 382 | 64.40% | 1.01
sympy indefinite integral | 119 | 193 | 61.66% | 1.01
sympy evaluate integral | 87 | 146 | 59.59% | 1.01
sympy solve integral | 69 | 103 | 66.99% | 1

So to the extent that searchers use a verb, it is _solve_. However, most searchers don't use a verb.